fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -441,7 +441,7 @@ ACTOR Future<Void> getRate(UID myID,
 			stats->batchTransactionRateAllowed = rep.batchTransactionRate;
 			++stats->updatesFromRatekeeper;
 			//TraceEvent("GrvProxyRate", myID).detail("Rate", rep.transactionRate).detail("BatchRate", rep.batchTransactionRate).detail("Lease", rep.leaseDuration).detail("ReleasedTransactions", *inTransactionCount - lastTC);
-			lastTC = *inTransactionCount;
+			// lastTC = *inTransactionCount;
```

Comment:
Meant to be deleted?

Reply:
No. The line above has a commented out TraceEvent that uses `lastTC`, so I just comment this out as well.